index.html

```diff
@@ -10,7 +10,25 @@
     <!-- Шапка сайта -->
     <header>Шапка сайта</header>
     <!-- Секция 1 -->
-    <section>Секция 1</section>
+    <section class="section" id="warranty">
+      <div class="wrapper">
+        <div class="text-block">
+          <h1 class="title">Peaceful shopping mindfull money</h1>
+          <p class="copy">See how we can help with making your shopping experience and money management more ZEN.</p>
+          <div class="links">
+            <a class="link" href="https://apple.com">
+              <img src="./assets/icons/apple.svg" alt="Apple icon">
+            </a>
+            <a class="link" href="https://google.com">
+              <img src="./assets/icons/google.svg" alt="Google icon">
+            </a>
+          </div>
+        </div>
+        <div class="image-block">
+          <img src="./assets/images/3.jpg" alt="Bag and device">
+        </div>
+      </div>
+    </section>
```

Comment:
Use id name "download"
Use BEM methodology
Use class name "download" for block